lib/edit_profile_page.dart

```diff
@@ -3,37 +3,43 @@ import 'package:cloud_firestore/cloud_firestore.dart';
 import 'main.dart'; //for currentuser & google signin instance
 import 'models/user.dart';
 import 'package:shared_preferences/shared_preferences.dart';
+import 'package:image_picker/image_picker.dart';
 
 class EditProfilePage extends StatelessWidget {
   final TextEditingController nameController = TextEditingController();
   final TextEditingController bioController = TextEditingController();
+  File _image;
+  String imagelink = "";
 
-  changeProfilePhoto(BuildContext parentContext) {
-    return showDialog(
-      context: parentContext,
-      builder: (BuildContext context) {
-        return AlertDialog(
-          title: Text('Change Photo'),
-          content: SingleChildScrollView(
-            child: ListBody(
-              children: <Widget>[
-                Text(
-                    'Changing your profile photo has not been implemented yet'),
-              ],
-            ),
-          ),
-        );
-      },
-    );
+
+  Future getImage() async {
+    var image = await ImagePicker.pickImage(source: ImageSource.gallery);
+
+    setState(() {
+      _image = image;
+      print('Image Path $_image');
+    });
+    uploadPic();
```

Comment:
why is this in a separate function? We don't reuse `uploadPic()`, so might as well just merge it with `getImage()`, and can rename `getImage()` to `changeProfilePicture()` to